signingscript/src/signingscript/sign.py

```diff
@@ -1551,13 +1554,20 @@ def verify_mar_signature(cert_type, fmt, mar, keyid=None):
     else:
         crosscert = None
 
+    if comment and orig_path.endswith('.msi'):
+        log.info(f"Using comment '{comment}' to sign {orig_path}")
+    elif comment:
+        log.info("Not using specified comment to sign {orig_path}, not yet implemented.")
```

Comment:
In general we should use %s for logging. e.g.
```log.info("Using comment %s to sign %s", comment, orig_path)```

This avoids problems where the f-string evalutation fails and your logging call ends up raising an exception and causing your program to crash.